docs/ancestry.md

```diff
@@ -2071,6 +2071,27 @@ print(tree.tmrca(0,1))
 
 ```
 
+Because branch lengths are proportional to {math}`N^2` generations, other events (such as mutations or recombinations) will also be proportional to {math}`N^2`. For example, if c is set to 0, the dirac coalescent will behave as if under a Moran model. However if believed census size is inputted (e.g. {math}`N=10000`), a number of segregating sites proportional to {math}`N^2` will be obtained. Leading to unrealistic simulated data.
```

Comment:
I don't think it's a good idea to introduce the concept of a Moran model here. We don't want to make this any worse than it has to be for people who are not familiar with obtaining coalescent processes as scaling limits from forward-time models. I also don't think that we should say any particular outcome is more or less realistic - that's highly application-specific. Instead, we should help practitioners calibrate what number they should plug in for `N` to get the results they expect. I've suggested a slight redraft below.

Because branch lengths are proportional to {math}`N^2` generations, the number of events typically simulated along branches (such as mutations or recombinations) will also be proportional to {math}`N^2`.
For example, if a tree sequence generated using a given census size {math}`N` is used as input for simulating segregating sites (see {ref}`Mutation simulations<sec_mutations>` for details on how to do this), the expected number of segregating sites obtained will be proportional to {math}`N^2`.

Reply:
Sorry, I'm really very late here - but, is the reason for branch lengths being `N^2` the same as for the Moran model? For the Moran model, it takes order of `N^2` *units of time* to coalesce, but it also takes `N` units of time for a typical individual to reproduce, so I'd argue that the time to coalescence in the Moran is still `N` generations (well, `2N`, I guess). So, if I were to implement the Moran model in msprime, I'd have chosen that time scaling, so that things like mutation rates match what they mean in other models (i.e., mutations per generation).

Reply:
@petrelharp, the reason for the `N^2` scaling for the Dirac-coalescent is exactly the same as it is for the Moran model, and you're totally right about the scaling. On the other hand, the reason for the `N^{a-1}` scaling for the Beta-coalescent is different, and can't be addressed by a similar "time units to generations" mapping. From memory, given that we were going to have to explain the latter anyway, we thought it would be less obscure to use `N^2` for the former as well (matching the time scaling in the source material we cite for the model), and avoid any mention of time units distinct from generations.

Reply:
ok, great - I figured you were thinking about this, but wanted to make sure.